modules/bemenu.nix

```
@@ -0,0 +1,34 @@
+{pkgs, config, lib, ... }:
+
+with config.lib.stylix.colors.withHashtag;
+with config.stylix.fonts;
+{
+  options.stylix.targets.bemenu = {
+    enable = config.lib.stylix.mkEnableTarget "bemenu" true;
+    fontSize = lib.mkOption { type = lib.types.str; default = ""; }; # optional argument
+    alternate = lib.mkOption { type = lib.types.bool; default = false; };
+  };
+
+  config.home-manager.sharedModules = lib.mkIf config.stylix.targets.mako.enable [{
```

Almost missed this one!

```suggestion
  config.home-manager.sharedModules = lib.mkIf config.stylix.targets.bemenu.enable [{
```